pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -71,12 +71,15 @@ func (s *Status) Code() Code {
 
 // Message returns message of the Status.
 func (s *Status) Message() string {
+	if s == nil {
+		return ""
+	}
 	return s.message
 }
 
 // IsSuccess returns true if and only if "Status" is nil or Code is "Success".
 func (s *Status) IsSuccess() bool {
-	if s == nil || s.code == Success {
+	if s.Code() == Success {
```

Comment:
Would `return s.Code() == Success` be better?

Reply:
Sounds good, done.